packages/client/vite.config.ts

```diff
@@ -171,7 +171,7 @@ export default defineConfig(async () => {
       mediapipe_workaround(),
       PkgConfig(),
       process.env.VITE_PWA_ENABLED === 'true' ? PWA(clientSetting) : undefined,
-      createEjsPlugin({
+      ViteEjsPlugin({
```

Comment:
This does not seem to be how the API is used. the `data` property should be the object passed in.